server-example/test_app.py

```diff
@@ -0,0 +1,13 @@
+import pytest
```

Comment:
Could you add a license to the top of this file? You can use any other file in the project as an example but make sure that you change the year to 2020.

Reply:
Also, every file needs a comment explaining what it is doing.